services/ui-src/src/components/pages/Dashboard/MobileDashboardTable.tsx

```diff
@@ -66,10 +67,14 @@ export const MobileDashboardTable = ({
             )}
           </Text>
         </Box>
-        {reportType === "WP" && (
+        {/* Admin: Submission count */}
```

Comment:
can this get removed?

Reply:
we decided in slack on letting the comment stay